highcharts_excentis/highcharts/highcharts.py

```diff
@@ -1,31 +1,43 @@
 # -*- coding: utf-8 -*-
 
-from __future__ import unicode_literals, absolute_import
-from future.standard_library import install_aliases
-install_aliases()
-from past.builtins import basestring
+from __future__ import absolute_import, unicode_literals
 
-from jinja2 import Environment, PackageLoader
+from future.standard_library import install_aliases
 
-import json, uuid
-import re
+install_aliases()
 import datetime
-import urllib.request, urllib.error, urllib.parse
 import html
-from .options import BaseOptions, ChartOptions, ColorAxisOptions, \
-    ColorsOptions, CreditsOptions, DrilldownOptions, ExportingOptions, \
-    GlobalOptions, LabelsOptions, LangOptions, \
-    LegendOptions, LoadingOptions, NavigationOptions, PaneOptions, \
-    PlotOptions, SeriesData, SubtitleOptions, TitleOptions, \
-    TooltipOptions, xAxisOptions, yAxisOptions, zAxisOptions, MultiAxis
+import json
+import re
+import urllib.error
+import urllib.parse
+import urllib.request
+import uuid
 
+import diskcache as dc
+from jinja2 import Environment, PackageLoader
+from past.builtins import basestring
+
+from .common import (ArrayObject, ColorObject, CommonObject, CSSObject,
+                     Formatter, JSfunction, Levels, RawJavaScriptText,
+                     SVGObject)
 from .highchart_types import Series, SeriesOptions
-from .common import Levels, Formatter, CSSObject, SVGObject, JSfunction, RawJavaScriptText, \
-    CommonObject, ArrayObject, ColorObject
+from .options import (BaseOptions, ChartOptions, ColorAxisOptions,
+                      ColorsOptions, CreditsOptions, DrilldownOptions,
+                      ExportingOptions, GlobalOptions, LabelsOptions,
+                      LangOptions, LegendOptions, LoadingOptions, MultiAxis,
+                      NavigationOptions, PaneOptions, PlotOptions, SeriesData,
+                      SubtitleOptions, TitleOptions, TooltipOptions,
+                      xAxisOptions, yAxisOptions, zAxisOptions)
 
 CONTENT_FILENAME = "content.html"
 PAGE_FILENAME = "page.html"
 
+# 50MB limit, to ensure LRU evection
+_CACHE = dc.Cache('./highcharts_cache', size_limit=50 * 1024**2)
```

Comment:
I would suggest to use a `.`-prefixed directory, this marks it as "hidden" in Unix-based OS and thus hints the user to not alter its contents. We can also include this package name to avoid overriding directories/file created by other packages.

For example:
```
./.cache/highcharts-excentis
```

**Note**: *It will need to be verified if directories including sub-directories are properly created.*

```suggestion
_CACHE = dc.Cache('./.cache/highcharts-excentis', size_limit=50 * 1024**2)
```